src/modules/Support/html_client/mod_support_ticket.html.twig

```diff
@@ -4,199 +4,213 @@
 
 {% block meta_title %}{{ ticket.subject }}{% endblock %}
 
+{% block head %}
+    {{ encore_entry_link_tags('markdown') }}
+{% endblock %}
+
 {% block body_class %}support-ticket{% endblock %}
 {% block breadcrumb %}
-<li><a href="{{ 'support' | link}}">{{ 'Support tickets'|trans }}</a><span class="divider">/</span></li>
-{{ticket.subject}}
+    <li class="breadcrumb-item"><a href="{{ 'support' | link }}">{{ 'Support tickets'|trans }}</a></li>
+    <li class="breadcrumb-item active" aria-current="page">{{ ticket.subject }}</li>
 {% endblock %}
 
 {% block content %}
-<div class="row">
-    <article class="span4 data-block">
-        <div class="data-container">
-            <header>
-                <h2>{{ 'Ticket information'|trans }}</h2>
-
-                {% if ticket.status != 'closed' %}
-                    <div class="data-header-actions">
-                        <button class="btn btn-small" type="button" id="ticket-close">{{ 'Close ticket'|trans }}</button>
+    <div class="row">
+        <div class="col-md-12">
+            <div class="card mb-3">
+                <div class="card-header py-3">
+                    <div class="d-flex justify-content-between align-items-center">
+                        <div>
+                            <h5 class="mb-2">{{ 'Ticket'|trans }} #{{ ticket.id }}</h5>
+                        </div>
+                        <div class="">
+                            <div class="d-flex gap-2">
+                                {% if ticket.status != 'closed' %}
+                                    <button class="btn btn-info btn-sm" id="reply-to-btn">{{ 'Reply'|trans }}</button>
+                                {% endif %}
+                                <button class="btn btn-secondary btn-sm" type="button"
+                                        id="ticket-close">{{ 'Close ticket'|trans }}</button>
```

Comment:
This needs to be removed if the ticket is closed. 

![grafik](https://github.com/FOSSBilling/FOSSBilling/assets/1939311/d4c1d4f4-153d-411b-8c52-32b21f7d45a0)